packages/react/src/primitives/types/style.ts

```diff
@@ -56,21 +56,21 @@ export interface BaseStyleProps extends FlexItemStyleProps, GridItemStyleProps {
   letterSpacing?: ResponsiveStyle<StyleToken<Property.LetterSpacing>>;
   lineHeight?: ResponsiveStyle<StyleToken<Property.LineHeight>>;
   margin?: ResponsiveStyle<StyleToken<Property.Margin>>;
-  marginBottom?: ResponsiveStyle<StyleToken<Property.MarginBottom>>;
-  marginLeft?: ResponsiveStyle<StyleToken<Property.MarginLeft>>;
-  marginRight?: ResponsiveStyle<StyleToken<Property.MarginRight>>;
-  marginTop?: ResponsiveStyle<StyleToken<Property.MarginTop>>;
+  marginBottom?: ResponsiveStyle<StyleToken<Property.MarginBlockEnd>>;
```

Comment:
Can you please also add all the logical margin and padding style props?
marginBlockEnd
marginInlineStart
marginInlineEnd
marginBlockStart

paddingBlockEnd
paddingInlineStart
paddingInlineEnd
paddingBlockStart